examples/rest/crypto-snapshots_all_tickers.py

```diff
@@ -0,0 +1,45 @@
+from polygon import RESTClient
+from polygon.rest.models import (
+    TickerSnapshot,
+    Agg,
+)
+
+# docs
+# https://polygon.io/docs/crypto/get_v2_snapshot_locale_global_markets_crypto_tickers
+# https://polygon-api-client.readthedocs.io/en/latest/Snapshot.html#get-all-snapshots
+
+# client = RESTClient("XXXXXX") # hardcoded api_key is used
+client = RESTClient()  # POLYGON_API_KEY environment variable is used
+
+snapshot = client.get_snapshot_all("crypto")  # all tickers
+
+# print raw values
+print(snapshot)
+
+# crunch some numbers
+for item in snapshot:
+
+    # verify this is an TickerSnapshot
+    if isinstance(item, TickerSnapshot):
+
+        # verify this is an Agg
+        if isinstance(item.prev_day, Agg):
+
+            # verify this is a float
+            if isinstance(item.prev_day.open, float) and isinstance(
```

Comment:
Why do you have to do all this type verification? I believe all of our responses are using type hinting. Do we not have a type hint linter to ensure types are correct?

Reply:
Yes, this was to satisfy the `poetry run black --check polygon test_* examples` check that happens. I originally didn't have any of this in here but the linter wanted to enforce all this logic. So, I added it to get the linting to pass.